src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -422,13 +422,6 @@ export default function Swap({
   // check if user has gone through approval process, used to show two step buttons, reset on token change
   const [approvalSubmitted, setApprovalSubmitted] = useState<boolean>(false)
 
-  // reset url query on network change
-  useEffect(() => {
-    if (chainId && previousChainId && chainId !== previousChainId) {
-      history.replace(location.pathname)
-    }
-  }, [chainId, history, location.pathname, previousChainId])
-
```

Comment:
This was kind of in a race condition, resetting the URL to /swap which then would cause the search parameter to be set again.
Doesn't seem to be needed anymore